diff --git b/scripts/buildbot_scripts/ICON.cfg a/scripts/buildbot_scripts/ICON.cfg
index df2c59fac9..3e7b30b0d6 100644
--- b/scripts/buildbot_scripts/ICON.cfg
+++ a/scripts/buildbot_scripts/ICON.cfg
@@ -15,6 +15,7 @@ import sys
 import os
 
 sys.path.append("py")
+sys.path.append("buildbot_scripts-buildbotlists")
 reload(sys)
 sys.setdefaultencoding('utf8')
 
@@ -47,8 +48,7 @@ c['properties'] = {'section' : 'all',
 	           'subdir' : 'default'
                   }
 
-c['slaves'] = [BuildSlave("daint_cpu" , "bot1passwd", notify_on_missing="help-buildbot@zmaw.de", missing_timeout=300)
-              ,BuildSlave("mistral" , "bot1passwd", notify_on_missing="help-buildbot@zmaw.de", missing_timeout=300)
+c['slaves'] = [BuildSlave("mistral_test" , "bot1passwd", notify_on_missing="karl-hermann.wieners@mpimet.mpg.de", missing_timeout=300)
               ]
 
 # to limit to two concurrent builds on a slave, use
@@ -56,7 +56,7 @@ c['slaves'] = [BuildSlave("daint_cpu" , "bot1passwd", notify_on_missing="help-bu
 
 # 'slavePortnum' defines the TCP port to listen on. This must match the value
 # configured into the buildslaves (with their --master option)
-c['slavePortnum'] = 9988
+c['slavePortnum'] = 9989
 
 
 ####### CHANGESOURCES
@@ -92,8 +92,9 @@ from buildbot.changes.gitpoller import GitPoller
 c['change_source'] = [
     PBChangeSource(),
     GitPoller(
-        repourl = 'git@git.mpimet.mpg.de:icon-nwp.git',
-        pollinterval = 1800,
+        repourl = 'git@git.mpimet.mpg.de:icon.git',
+        project='ICON',
+        pollinterval = 1800
     )
 ]
 
@@ -113,16 +114,11 @@ from buildbot.schedulers.forcesched \
 # Definition of the defaults =========
 #default_Branch     = 'master'
 default_builder_list = [
-    'DAINT_CPU_cce',
     'MISTRAL_intel_hybrid', 'MISTRAL_intel', 'MISTRAL_intel_openmp',
     'MISTRAL_ocean',
     'MISTRAL_gcc',
     'MISTRAL_nag', 'MISTRAL_nag_serial'
 ]
-nightly_builder_list = [
-    'MISTRAL_intel_hybrid', 'MISTRAL_intel', 'MISTRAL_intel_openmp',
-    'MISTRAL_gcc', 'MISTRAL_nag', 'MISTRAL_nag_serial'
-]
 
 # Test changes on the default branch whenever they occur
 # NOTE: currently disabled!
@@ -137,9 +133,8 @@ s1 = AnyBranchScheduler(
 n1 = scheduler.Nightly(
          name = 'nightly_nwp',
          properties = {'subdir' : 'nightly_nwp'},
-         builderNames = nightly_builder_list,
-         codebases = {'': {'repository': 'git@git.mpimet.mpg.de:icon-nwp.git'}},
-         branch = 'icon-nwp/icon-nwp-dev',
+         builderNames = default_builder_list,
+         branch = 'branches/icon-nwp/icon-nwp-dev',
          hour = 01,
          minute = 00,
          onlyIfChanged = True,
@@ -179,7 +174,7 @@ f1 = ForceScheduler(
          ),
      )
 
-c['schedulers'] = [f1, n1]
+c['schedulers'] = [f1]
 
 
 # ================================================================
@@ -427,7 +422,7 @@ clean_workdir = ICON_Del(
 # create working copy
 vcs_update = source.Git(
     mode              = 'update',
-#   repourl           = 'git@git.mpimet.mpg.de:icon-nwp.git',
+#   repourl           = 'git@git.mpimet.mpg.de:icon.git',
     description       = "git",
     workdir           = work_dir,
 #   branch            = default_Branch,
@@ -489,30 +484,26 @@ m2.addStep(make_doc)
 #========================================================
 # Builders/BuildSlave definitions
 c['builders'] = [
-    {'name':'DAINT_CPU_cce',
-        'slavename':'daint_cpu', 'builddir':'daint_cpu_cce', 'factory':run_full,
-        'env':{'BB_SYSTEM':'daint_cpu', 'BB_SLAVE':'cce'}},
-
     {'name':'MISTRAL_intel_hybrid',
-        'slavename':'mistral', 'builddir':'mistral_intel_hybrid', 'factory':run_full,
+        'slavename':'mistral_test', 'builddir':'mistral_intel_hybrid', 'factory':run_full,
         'env':{'BB_SYSTEM':'mistral' , 'BB_SLAVE':'intel_hybrid'}},
     {'name':'MISTRAL_intel',
-        'slavename':'mistral', 'builddir':'mistral_intel', 'factory':run_full,
+        'slavename':'mistral_test', 'builddir':'mistral_intel', 'factory':run_full,
         'env':{'BB_SYSTEM':'mistral' , 'BB_SLAVE':'intel'}},
     {'name':'MISTRAL_intel_openmp',
-        'slavename':'mistral', 'builddir':'mistral_intel_openmp', 'factory':run_full,
+        'slavename':'mistral_test', 'builddir':'mistral_intel_openmp', 'factory':run_full,
         'env':{'BB_SYSTEM':'mistral' , 'BB_SLAVE':'intel_openmp'}},
     {'name':'MISTRAL_ocean',
-        'slavename':'mistral', 'builddir':'mistral_ocean', 'factory':run_full,
+        'slavename':'mistral_test', 'builddir':'mistral_ocean', 'factory':run_full,
         'env':{'BB_SYSTEM':'mistral' , 'BB_SLAVE':'ocean'}},
     {'name':'MISTRAL_gcc',
-        'slavename':'mistral', 'builddir':'mistral_gcc', 'factory':run_full,
+        'slavename':'mistral_test', 'builddir':'mistral_gcc', 'factory':run_full,
         'env':{'BB_SYSTEM':'mistral' , 'BB_SLAVE':'gcc'}},
     {'name':'MISTRAL_nag',
-        'slavename':'mistral', 'builddir':'mistral_nag', 'factory':run_full,
+        'slavename':'mistral_test', 'builddir':'mistral_nag', 'factory':run_full,
         'env':{'BB_SYSTEM':'mistral' , 'BB_SLAVE':'nag'}},
     {'name':'MISTRAL_nag_serial',
-        'slavename':'mistral', 'builddir':'mistral_nag_serial', 'factory':run_full,
+        'slavename':'mistral_test', 'builddir':'mistral_nag_serial', 'factory':run_full,
         'env':{'BB_SYSTEM':'mistral' , 'BB_SLAVE':'nag_serial'}}
 ]
 #========================================================
@@ -563,7 +554,7 @@ authorizations = Authz(
 )
 
 WEB = html.WebStatus(
-    http_port = "tcp:8011:interface=127.0.0.1",
+    http_port = "tcp:9011:interface=127.0.0.1",
     ### allowForce = True,
     authz = authorizations
 )
@@ -581,7 +572,7 @@ from buildbot.status import mail
 # Any failures
 # ------------
 
-c['status'].append(mail.MailNotifier(fromaddr="help-buildbot@zmaw.de",
+c['status'].append(mail.MailNotifier(fromaddr="buildbot@zmaw.de",
                                      mode='failing',
                                      subject='[BUILDBOT ICON] %(builder)s BUILD FAILED',
                                      relayhost="smtp.zmaw.de",
@@ -593,7 +584,7 @@ c['status'].append(mail.MailNotifier(fromaddr="help-buildbot@zmaw.de",
 
 from buildbot.status import client
 c['status'].append(
-    client.PBListener(port = 9998)
+    client.PBListener(port = 9981)
 )
 
 
@@ -635,6 +626,6 @@ c['projectURL']  = "https://code.zmaw.de/projects/icon"
 # with an externally-visible host name which the buildbot cannot figure out
 # without some help.
 
-c['buildbotURL'] = "http://buildbot.zmaw.de/icon/"
+c['buildbotURL'] = "http://buildbot.zmaw.de/icon_test/"
 
 # vim:ft=python
